sql/ddl.sql

```diff
@@ -769,6 +894,17 @@ COMMENT ON COLUMN punkt.registreringtil IS 'Tidspunktet hvor en ny registrering
 COMMENT ON COLUMN punkt.sagseventfraid IS 'Angivelse af den hændelse der har bevirket registrering af et fikspunktsobjekt';
 COMMENT ON COLUMN punkt.sagseventtilid IS 'Angivelse af den hændelse der har bevirket afregistrering af et fikspunktsobjekt';
 
+COMMENT ON TABLE punktsamling IS 'Gruppering af Punkter i samlinger, typisk i forbindelse med overvågning af punkters stabilitet.';
```

Comment:
Ville det vive mening at bemærke, at konceptet punktsamling er forskelligt fra konceptet punktgruppe, som for en landmåler/landinspektør betyder noget lidt andet?

Alternativt (eller supplerende) kan dette måske fremhæves andre steder end i databasen, hvor det nok ikke læses af så mange. Det er bare en tanke.

Reply:
Ja, det giver nok mere mening at udpensle det bedre i dokumentationen et sted. Det kan evt. være i datamodelsbeskrivelsen men det passer måske endnu bedre ind i noget brugervendt dokumentation som kommer på et senere tidspunkt?

Reply:
Jep.